### DISCUSSION TITLE
Admin backend is broken with GVFS 1.48.0

### DISCUSSION THREAD

**Discussion by tsujan** [lxqt/lxqt#1970] [General]
Arch temporarily removed it from its package `gvfs-1.48.0-2`. The reason seems acceptable to me: https://bugs.archlinux.org/task/70108.

It may have not been removed by other distros (I checked Fedora) but `admin:///` doesn't work for them either.

I opened this page to explain that the problem is not ours; it should be fixed in GVFS (by Gnome devs).

**Comment by tsujan** [comment]
The problem is fixed upstream. Manjaro's packages "1.48.0+13+gc615cfe1-1" have the fix.

**Reply by stefonarch** [reply]
It does work for you now? On arch I get "impossibile to open as admin" error still.

**Reply by tsujan** [reply]
Arch doesn't have the fix yet. Manjaro added it independently of Arch.

**Reply by stefonarch** [reply]
I see, I thought there were always _after_ but that's not true :)

**Reply by tsujan** [reply]
It surprised me too. Arch maintainers had reported the bug.
